CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file based on the
 - Remove Elastica/Query/Image.php and test/Elastica/Query/ImageTest.php, no more support for image-plugin.
 - Remove Elastica/Query/Filtered.php and test/Elastica/Query/FilteredTest.php and all uses from code.
 - Remove index.merge.policy.merge_factor, and set/get MergePolicy as it looks deprecated from ES 1.6
+- Remove all deprecated method and classes for the new release.
```

Comment:
Even though the list is going to be quite long, we probably have to be here more specific for people that upgrade and list all the methods that were removed. Please also link to the PR.